test/prism_regression/regexp.rb

```diff
@@ -0,0 +1,9 @@
+# typed: true
+
+/foo/
```

Comment:
Let's check for null issues w.r.t. empty:

```suggestion
// # empty Regexp

/foo/
```

(Idk why anybody would ever want an empty regex that matches everything, but 🤷‍♂️)